### PR TITLE
[SYCL][XPTI][Doc] PI -> UR renaming

### PR DESCRIPTION
**Pull request by martygrant**
Closes https://github.com/intel/llvm/issues/15109

### CODE REVIEW

File: sycl/source/detail/xpti_registry.hpp [hunk 1]
```diff
@@ -27,11 +27,6 @@ namespace detail {
 // We define a sycl stream name and this will be used by the instrumentation
 // framework
 inline constexpr const char *SYCL_STREAM_NAME = "sycl";
-// Stream name being used for traces generated from the SYCL plugin layer
-inline constexpr const char *SYCL_PICALL_STREAM_NAME = "sycl.pi";
-// Stream name being used for traces generated from UR calls. This stream
-// contains information about function arguments.
-inline constexpr const char *SYCL_PIDEBUGCALL_STREAM_NAME = "sycl.pi.debug";
```
Comment (martygrant) [inline sycl/source/detail/xpti_registry.hpp:34]: These were to be renamed but I didn't see any usage of them throughout the project. Can someone let me know if they are used in some other way and should be kept and renamed.

**Comment by steffenlarsen** [comment]
This looks reasonable to me, but I trust @KseniyaTikhomirova and @tovinkere more than myself when it comes to XPTI.

File: sycl/source/detail/xpti_registry.hpp (hunk 1 above)
Comment (KseniyaTikhomirova) [reply sycl/source/detail/xpti_registry.hpp:34]: I think it was removed during pi plugin removal. These were streams for pi calls notifications.

File: xptifw/samples/syclur_collector/README.md [hunk 2]
```diff
@@ -19,9 +19,9 @@ an application instrumented with XPTI, the following steps must be performed.
    `XPTI_FRAMEWORK_DISPATCHER=/path/to/libxptifw.[so,dll,dylib]`
 
 3. Set the environment variable that points to the subscriber, which in this
-  case is `libsyclpi_collector.[so,dll,dylib]`.
-
-     `XPTI_SUBSCRIBERS=/path/to/libxpti_syclpi_collector.[so,dll,dylib]`
+  case is `libsyclur_collector.[so,dll,dylib]`.
 
+     `XPTI_SUBSCRIBERS=/path/to/libxpti_syclur_collector.[so,dll,dylib]`
+CHECK THIS!!!
```
Comment (KseniyaTikhomirova) [inline xptifw/samples/syclur_collector/README.md:25]: comment to be removed I guess

File: xptifw/samples/syclur_collector/syclur_collector.cpp [hunk 3]
```diff
@@ -32,15 +32,15 @@ XPTI_CALLBACK_API void xptiTraceInit(unsigned int major_version,
                                      const char *stream_name) {
   // The example basic collector under xptifw/samples/basic_collector takes in
   // streams from anyone as an example. In this collector, we will accept
-  // streams from just the SYCL  plugin interface (PI) layer.
+  // streams from just the SYCL Unified Runtime (UR) layer.
   printf("Stream Name: %s\n", stream_name);
   if (std::string("sycl.pi") == stream_name) {
```
Comment (KseniyaTikhomirova) [inline xptifw/samples/syclur_collector/syclur_collector.cpp:37]: sycl.pi is not the case any more

File: xptifw/samples/syclur_collector/README.md (hunk 2 above)
Comment (martygrant) [reply xptifw/samples/syclur_collector/README.md]: Removed, thanks.

File: xptifw/samples/syclur_collector/syclur_collector.cpp (hunk 3 above)
Comment (martygrant) [reply xptifw/samples/syclur_collector/syclur_collector.cpp]: Thanks! I went ahead and done a full search and replace for `sycl.pi` to `ur.call` throughout the project.

File: sycl/source/detail/xpti_registry.hpp (hunk 1 above)
Comment (martygrant) [reply sycl/source/detail/xpti_registry.hpp:34]: Thanks good to know they aren't used.

**Review by KseniyaTikhomirova — APPROVED** [review]

**Comment by martygrant** [comment]
@intel/llvm-gatekeepers ready for merging please

**Pull request merged**